charts/mgr-tenants/values.yaml

```
@@ -0,0 +1,158 @@
+# Default values for mgr-tenants.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  repository: 732722833398.dkr.ecr.us-west-2.amazonaws.com732722833398.dkr.ecr.us-west-2.amazonaws.com/mgr-tenants
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "latest"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+replicaCount: 1
+
+resources:
+  limits:
+    # cpu: 100m
+    memory: 2Gi
+  requests:
+    # cpu: 50m
+    memory: 256Mi
+
+autoscaling:
+  enabled: false
+  # minReplicas: 1
+  # maxReplicas: 4
+  # targetCPUUtilizationPercentage: 80
+  # targetMemoryUtilizationPercentage: 80
+
+# Should be set to NodePort if ingress for AWS ALB enabled
+service:
+  type: ClusterIP
+  ports:
+    - name: http
+      protocol: TCP
+      port: 80
+      containerPort: 8081
+      targetPort: http
+  annotations: {}
+
+ingress:
+  enabled: false
+  className: ""
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  hosts:
+    - host: chart-example.local
+      paths:
+        - path: /*
+          pathType: ImplementationSpecific
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+args: []
+## @param javaOpts Custom JVM parameters
```

Shouldn't we enable Eureka here?
```suggestion
eureka:
  enabled: true
```

---

NOTE: mgr-* mgmt components do not require sidecar to be inside a pod (standalone)